docs/codeathon.md

```diff
@@ -0,0 +1,25 @@
+# FAIR Codeathon 2021 Helpful Information
+
+![codeathon](https://images.ctfassets.net/6bya4tyw8399/2qgsOmFnm7wYIfRrPrqbgx/ae3255858aa12bfcebb52e95c7cacffe/codeathon-graphic.png)
+
+For participants of the [FAIR Codeathon](https://sparc.science/help/2021-sparc-fair-codeathon) July 12-27 2021, we have compiled the parts of the documentation that are likely to be most useful to you. As a very brief introduction, 
+
+**o<sup>2</sup>S<sup>2</sup>PARC** is a cloud-based computational platform for running code that is contained in various modules, called [services](docs/platform_introduction/core_elements/services.md). These services can be chained together which allows for file-based information transfer (see [Connecting Services](docs/study_setup/connecting_services.md)) from one service to another. A project on **o<sup>2</sup>S<sup>2</sup>PARC**, which contains these chained pipelines is called a [study](docs/platform_introduction/core_elements/studies.md).
```

Comment:
I understand that you want to underline file transfer but as you know it can also transfer basic types. Perhaps a footnote ?

Reply:
done